### PR TITLE
[#46] Implemented error handling in Form based on API responses

### PR DESCRIPTION
**Pull request by Monstarrrr**

### CODE REVIEW

**Comment by purple-void** [comment]
- Logs errors into the console
- Removed confirmation password
- Fixed error handling from [feat/1_auth](https://github.com/Monstarrrr/rebutify/tree/feat/1_auth)
- Fixed errors disappearing after registering user

File: frontend/app/(routes)/(auth)/login/page.tsx [hunk 1]
```diff
@@ -28,22 +29,26 @@ export default function Login() {
     const password = formData.get('password')
 
     try {
-      const { data } = await api.post('auth/token/login', {
+      const { data } = await api.post('/auth/jwt/create', {
         password,
         username,
       })
       console.log('# data :', data)
```
Comment (purple-void) [inline frontend/app/(routes)/(auth)/login/page.tsx:36]: do you mean to keep this console log?

File: frontend/app/(routes)/(auth)/login/page.tsx [hunk 2]
```diff
@@ -28,22 +29,26 @@ export default function Login() {
...
-    } catch (error) {
+    } catch (error: any) {
+      const { response } = error
+      setApiErrors(response)
       console.log('# error :', error)
```
Comment (purple-void) [inline frontend/app/(routes)/(auth)/login/page.tsx:40]: do you mean to keep this console log?

File: frontend/app/(routes)/(auth)/login/page.tsx (hunk 1 above)
Comment (Monstarrrr) [reply frontend/app/(routes)/(auth)/login/page.tsx:36]: Missed them, will remove, thank you

File: frontend/app/(routes)/(auth)/login/page.tsx (hunk 2 above)
Comment (Monstarrrr) [reply frontend/app/(routes)/(auth)/login/page.tsx:40]: Missed them, will remove, thank you

**Comment by purple-void** [comment]
Looks good

**Pull request merged**